test/serial/logger.1.test.js

```diff
@@ -859,4 +859,60 @@ t.test('test log stream', (t) => {
 
     t.same(lines[0].req, {})
   })
+
+  t.test('Should throw an error if logger instance is passed to `logger`', async (t) => {
```

Comment:
Maybe add one when we use only `logger` to create the logger?

Reply:
what do you mean?

Reply:
A test scenario where we pass the `logger` property with options to build a logger instance